docs/usage.rst

```diff
@@ -489,7 +488,9 @@ batchmakeshell.py
    and clone all the subcontent except the content inside the 
    'objects' folder into them. The shells will be created into the
    targeted output path.
--  Usage: ``batchmakeshell.py $input -o $output``
+-  Usage: ``batchmakeshell.py $path/to/batch_directories -o /path/to/destination``
```

Comment:
Dollar sign variable symbol not used elsewhere in docs.

Reply:
Fixed here thanks!